docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -103,13 +105,15 @@ this yields us the following graph of computations
   />
 </center>
 
-##### Op Level Retries
+### Op-level retries
 
-In the tutorial DAG, the `t2` operator allowed for 3 retries. To configure the same behavior in Dagster, you can use [op level retry policies](/concepts/ops-jobs-graphs/op-retries)
+In the tutorial DAG, the `t2` operator allowed for 3 retries. To configure the same behavior in Dagster, you can use [op-level retry policies](/concepts/ops-jobs-graphs/op-retries)
 
-### Step 2: Defining the Job
+---
 
-In Dagster, the computations defined in `op`s are composed in `job`s, which define the sequence and dependency structure of the computations you want to execute. This directly corresponds to a DAG in Airflow. Here, we compose the `op`s `print_date`, `sleep` and `templated` to match the dependency structure defined by the Airflow operators `t1`, `t2`, and `t3`.
+## Step 2: Define the job
+
+In Dagster, the computations defined in ops are composed in jobs, which define the sequence and dependency structure of the computations you want to execute. This directly corresponds to a DAG in Airflow. Here, we compose the `op`s `print_date`, `sleep` and `templated` to match the dependency structure defined by the Airflow operators `t1`, `t2`, and `t3`.
```

Comment:
```suggestion
In Dagster, the computations defined in ops are composed in jobs, which define the sequence and dependency structure of the computations you want to execute. This directly corresponds to a DAG in Airflow. Here, we compose the op's `print_date`, `sleep` and `templated` to match the dependency structure defined by the Airflow operators `t1`, `t2`, and `t3`.
```

Can we remove the formatting around **op** here? It looks a bit odd when rendered, and there's no need to use code here